README.md

```diff
@@ -1,2 +1,39 @@
-# operator-iq
-IBM Operator for Nexus IQ
+# Sonatype Nexus IQ Certified Operator
+Red Hat certified OpenShift Operator for installing Sonatype Nexus IQ Server
+to an OpenShift cluster.
+
+## Building from Source for Local Development and Testing
+
+To develop and test locally, you'll use CodeReady Containers on your workstation
+and push your operator image to quay.io to make it available for installation.
+
+1. Install [https://developers.redhat.com/products/codeready-containers/overview](CodeReady Containers)
+   for a local Openshift 4 environment.
+2. Ensure you have a personal quay.io account.
+3. Build and deploy the operator image to your personal quay.io repository:
+   1. `operator-sdk build quay.io/<username>/nxiq-operator-certified`
```

Comment:
maybe not the question you want at this stage, but is `nxiq` the right name? We just had the discussion that `nxrm` doesn't mean anything outside of Sonatype and maybe the `nx` here suffers from the same issue. Possibly `nexusiq` or `nexus-iq` or `nexus-lifecycle`?

Reply:
this is the unique "project name". There are friendlier display names shown to the user, as you can see in the screenshots.

Reply:
it's just docs, so pretty easy to change this to something a bit more standard. Easy change to make it a little easier to follow